types/ramda/index.d.ts

```diff
@@ -1605,6 +1605,8 @@ export function pickAll(names: readonly string[]): <T, U>(obj: T) => U;
 /**
  * Returns a partial copy of an object containing only the keys that satisfy the supplied predicate.
  */
+export function pickBy(pred: ObjPred, record: Record<keyof any, any>): Record<keyof any, any>;
```

Comment:
ObjPred takes a defaulted generic, you should use it.

Reply:
Yes, using type parameters in this case will make type stronger, but it breaks the forward compatibility:
The previous type signature forced the user to specify type parameters implicitly. In order to make type overload work better and not break the already existing use case, I design the new signature to not use type parameter.

Reply:
Could you clarify what is the issue with the previous signatures and how the new ones are an improvement? I am still confused after reading the code and the discussions.